packages/thumbprint-global-css/src/packages/a.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../node_modules/@thumbtack/thumbprint-tokens/dist/scss/_index';
+
+a {
+    color: $tp-color__blue;
+    fill: $tp-color__blue;
```

Comment:
I suspect we don't need this. Was added for legacy svg icons that need a `fill` value to inherit color. We're using `color` now.

Reply:
Feel free to remove or leave if you're unsure.

Reply:
Will leave in for now. Can check later if it's really required.